JmeterNamesTemplater v1.1 and UtilityJmeterFunctions v1.1.3

DanilGolikov opened the pull request:
Created [JmeterNamesTemplater](https://github.com/DanilGolikov/JmeterNamesTemplater) that allows you to rename items according to the config file

undera commented on site/dat/repo/various.json:
```diff
@@ -2099,5 +2099,29 @@
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
+    "componentClasses": [
+      "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
```
`com.example.jmeter.plugin` does not look like a good namespace for plugin. Can you come up with something more distinctive? Something more author-specific.
Should be a simple refactoring.

undera commented on site/dat/repo/various.json:
```diff
@@ -2099,5 +2099,29 @@
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
```
`https://github.com/DanilGolikov/Custom_jmeter_functions` should be simpler and sufficient URL here.

Also I'd work on that page for better presentation, titles and help document structure. Up to you, though.

DanilGolikov commented:
Hi! I corrected the namespace, I didn't pay attention to the fact that it looks stupid xD before.
I also added another plugin (CustomJmeterFunctions), with your recommendations

DanilGolikov reviewed (commented):
.

undera commented on site/dat/repo/various.json:
```diff
@@ -2099,5 +2099,47 @@
+  },
+  {
+    "id": "CustomJmeterFunctions",
+    "name": "Custom Jmeter Functions",
```
There is existing plugin with this name, you need to choose a different name. Preferably a name that explains what kind of functions are inside. Like "Functions for strings templating" or like that.
Don't forget to change the ID of plugin, and also the description of it should explain more what is inside.

undera commented:
Please make the title of PR to reflect what is inside. Look at other PRs to get the idea

undera reviewed (changes requested):
Please address my comments

undera commented:
Looks like your JSON violate descriptor schema: https://github.com/undera/jmeter-plugins/actions/runs/10546658418/job/29218873882?pr=665

undera commented:
Looks good now
